src/components/ProductList.jsx

```
@@ -19,14 +19,23 @@ function ProductList() {
 
   const handleDelete = async (id) => {
     // TODO: DELETE API를 호출하고 fetchProducts() 호출
+    const response = await axios.delete(`${API_URL}/${id}`);
```

ESLint

[error] 'response' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)